R/tar_utils.R

```diff
@@ -0,0 +1,58 @@
+
+load_clipped_hydrobasins <- function(gdb,lyr_names, mask){
```

Comment:
so the input of `load_clipped_hydrobasins()` is the gdb containing the global hydrobasins (on gdrive). It then selects the relevant levels (levels 4 & 5) for the analysis and clips to nga admin 0. So the only way it would re-run is if: a.) the code in the function changes, b. global hydrobasin files change (they won't any time soon), or c. nga admin changes.

the resulting target object, `basins_clipped`, can be loaded w/ `tar_load(basins_clipped)` (same w/ all targets). This is nice if you wanted to use it for other code/visuals and it would theoretically get updated when any code/inputs changed.

The reason I named it `load_...()` is that it is loading from a file on gdrive rather than an object in memory. 

I think `load_hydrobasins()` makes sense